src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Actions from '../Actions';
+
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
+  return props => {
+    return <button className="Dropdown" onClick={() => props.onClick()} />;
+  };
+});
+
+jest.mock('../../Group', () => {
+  return  () => {
+    return <div className="Group" />;
+  };
+});
+
+describe('<Actions />', () => {
```

Comment:
```suggestion
describe('Actions', () => {
```

Reply:
fixed in "fixes after code review" commit